### ISSUE TITLE
instanceof doesn't work when using different kind of imports

### ISSUE THREAD

**Issue by nsine** [dividab/tsconfig-paths#119]
Hey there 👋

### TLDR;
I have created a small project to show the issue: https://github.com/nsine/tsconfig-paths-bug-example 🙂

### Description:
- In file `errors/classes.ts` we define custom error class `AppError` which extends `Error`
- And we throw it in `moreCode/code.ts` file in function `runCode`
- `errors/errorHanlder` runs this function in `try/catch` and here is the issue:

If i'm trying to check if `err instanceof AppError` - i will get `false`, however i used this class when throwing. After that i initialize the Error line after and it has correct `constructor` and therefore `instanceof` works there.

Looking at produced compiled js I see that file `errorHandler.js` imports `var classes_1 = require("./classes");` however `code.ts` has another import path `var classes_1 = require("~/errors/classes");` (I expect it's correct behavior). But looking at this `instanceof` example I assume that these two imports resolve module `classes.ts` twice as two different modules which is incorrect 👯‍♂️

Will appreaciate any ideas how to fix that!
